backend/src/modules/cards/dto/useCase/response/create-card-res.use-case.dto.ts

```diff
@@ -0,0 +1,13 @@
+import { ApiProperty } from '@nestjs/swagger';
+import { IsNotEmpty } from 'class-validator';
+import Card from 'src/modules/cards/entities/card.schema';
+
+export default class CreateCardResUseCaseDto {
```

Comment:
```suggestion
export default class CardCreationPresenter {
```


Great job in doing this. The objects that are returned to the client from the controller are called "presenters" and the dto are the opposite. So I suggest adding a new folder in the card's folder called "presenters" and then move this class there

Reply:
Yes, I noticed the return type is from the use case to the controller, but I think it should also be a presenter